pom.xml

```diff
@@ -307,7 +307,6 @@
 						<arg>-Xlint</arg>
 						<arg>-Xlint:-processing</arg>
 						<arg>-Xlint:-serial</arg>
-						<arg>-Werror</arg>
```

Comment:
This was added as a stopgap for the errors (from a dependency clash/incompatibility) on build reported in the team with updating `private-api-sdk` to a version > 2.0.109. Build is successful and this can be removed down the line once those issues are resolved

Reply:
Done